decision_tree_1:
  - id: 1
    variable: match_progress
    condition: == 4
    left_id: 2
    right_id: 3
    action: ""
  - id: 2
    variable: match_remainder
    condition: "> 300"
    left_id: -1
    right_id: 5
    action: ""
  - id: 3
    variable: ""
    condition: ""
    left_id: 4
    right_id: 4
    action: progress!=4
  - id: 4
    variable: ""
    condition: ""
    left_id: 6
    right_id: -1
    action: 111
  - id: 5
    variable: ""
    condition: ""
    left_id: 6
    right_id: -1
    action: stay
  - id: 6
    variable: ""
    condition: ""
    left_id: -1
    right_id: -1
    action: charge